nautobot/extras/models/models.py

```diff
@@ -551,6 +551,7 @@ class FileAttachment(BaseModel):
     filename = models.CharField(max_length=CHARFIELD_MAX_LENGTH)
     mimetype = models.CharField(max_length=CHARFIELD_MAX_LENGTH)
 
+    is_static_group_associable_model = False
```

Comment:
Should this and FileProxy be contact associable?

Reply:
Probably not.